data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -25,7 +25,7 @@ done
 # Start the provisioning nic if not already started
 PROVISIONING_NIC=ens4
 if ! nmcli -t device | grep "$PROVISIONING_NIC:ethernet:connected:provisioning"; then
-    nmcli c add type ethernet ifname $PROVISIONING_NIC con-name provisioning ip4 {{.PlatformData.BareMetal.ProvisioningIP}}/{{.PlatformData.BareMetal.ProvisioningCIDR}}
```

Comment:
Unless we're going to squash this commit should probably have a slightly more descriptive commit message (or at least prefix with baremetal: